packages/data/walkthroughs/9.9.9.json

```diff
@@ -1726,6 +1726,12 @@
         }
       ],
       "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P17",
+          "answerValue": "false",
+          "nextNavigateTo": "R20"
```

Comment:
I believe that this has to go after 

          "nextLogicType": "and",
          "valuesToCheck": [
            {
              "nextLogicType": "equals",
              "answerToCheck": "P15",
              "answerValue": "true"
            },
            {
              "nextLogicType": "equals",
              "answerToCheck": "P17",
              "answerValue": "false"
            }
          ],
          "nextNavigateTo": "P20"
        },

Otherwise, it won't handle this case correctly.

Reply:
We could also check if p15 is undefined instead.

Reply:
I'm remembering that I needed to ask the P15 questions first to account for the fact that accounting for the fact that P15 goes straight to L14.

I actually think here's what the logic would be after looking at it again.

- If p15 = true and  p17 == true go to p18 (skipping L13 logic)
- If p15 = true and  p17 == false go to p20 (skipping L13 logic)
- ~If p17 == No/False go to r20~
- If P5 or P8 contains true and  p17 == true go to p18
- ~If P5 or P8 contains true and p17 == false go to p20~
- Else Go to R20

Reply:
After discussing we realized the logic is the same and that it can be further simplified. Here's the final:

1. P17 === false && P15 === true ---> P20
2. P17 === true && (P15 === true || P5/P8 contain true) ---> P18
3. else R20

Reply:
Also I agree with your assessment that closer to the document is probably better in most cases so other people can understand our logic.

Reply:
Updated!